src/Functions/test/Functions.Scenario.Tests.ps1

```diff
@@ -0,0 +1,286 @@
+$TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzFunctionApp.Recording.json'
+$currentPath = $PSScriptRoot
+while(-not $mockingPath) {
+    $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+    $currentPath = Split-Path -Path $currentPath -Parent
+}
+. ($mockingPath | Select-Object -First 1).FullName
+
+
+# Test variables
+$testSubscriptionId = ""
+
+$resourceGroupNameWindowsPremium = "Func99-West-Europe-Win-Premium"
+$resourceGroupNameLinuxPremium = "Func99-West-Europe-Linux-Premium"
+$resourceGroupNameLinuxConsumption = "Func99-Southeast-Asia-Linux-Consumption"
+$resourceGroupNameWindowsConsumption = "Func99-Central-US-Windows-Consumption"
+
+$premiumPlanLocation = "West Europe"
+$linuxConsumptionLocation = "Southeast Asia"
+$windowsConsumptionLocation = "Central US"
+$planNameWorkerTypeLinux = "Functions-West-Europe-Linux"
+$planNameWorkerTypeWindows = "Functions-West-Europe-Windows"
+$storageAccountWindows = "functionswinstorage"
+$storageAccountLinux = "functionslinuxstorage"
+
+function CreateFunctionApps
+{
+    # Create resource groups
+    Write-Host "Creating resource groups" -ForegroundColor Green
+    New-AzResourceGroup -Name $resourceGroupNameWindowsPremium -Location $premiumPlanLocation -Force
```

Comment:
Generally, inside a test you either want to assign these to null or (better) pipe them through to a `Should` assertion command

Reply:
Yes, that makes sense. However, whenever I ran the Az.Functions module, I cannot run any other cmdlets that are part of Az.Accounts. What I mean is that I cannot create the resource groups within the tests. This has to be done on a separate PowerShell session. Is this expected?